### ISSUE TITLE
What is the last commit suitable for Android 11 MIUI v12.5.6?

### ISSUE THREAD

**Issue by mkcs121** [onettboots/kernel_xiaomay_arrow#11]
I compiled the kernel source code last month which is bootable successfully in Android 11. However, I found there is a minor problem or bug leading me to think about older versions. What would you suggest the last commit suitable for Raphael devices running Android 11 MIUI v12.5.6?

**Comment by onettboots** [comment]
latest updated for miui is 14-Miui branch

**Comment by mkcs121** [comment]
> latest updated for miui is 14-Miui branch

I used the build script provided to compile the kernel and flashed the anykernel zip file in TWRP recovery. However, it says "MIUI is not supported. Aborting..."

When I repack Image.gz-dtb into my stock boot image and flash, ver.1.9 stucks at the boot screen, while ver1.8 has some problems with v1 signature signing that any apk signers would crash when v1 signature scheme is included

**Comment by onettboots** [comment]
yes its caused anykernel for aosp version just edit update-binary file on META-INF, and drop "Miui" related and edit too on anykernel.sh for suported version to 10-15

**Comment by mkcs121** [comment]
> yes its caused anykernel for aosp version just edit update-binary file on META-INF, and drop "Miui" related and edit too on anykernel.sh for suported version to 10-15

How about the v1 signature scheme? I tested from bool-x 1.5, all apk signers would crash when v1 scheme is included
